src/tests/JIT/HardwareIntrinsics/Arm/Shared/Helpers.cs

```diff
@@ -5262,6 +5338,82 @@ public static double MultiplyExtended(double op1, double op2)
             }
         }
 
+        public static double FPExponentialAccelerator(ulong op1)
+        {
+            ulong index = op1 & 0b111111;
```

Comment:
I believe you are using `N == 16` in https://docsmirror.github.io/A64/2023-06/shared_pseudocode.html#impl-aarch64.FPExpA.1?

Reply:
For `N=32` and `N=64`, the index is the first 6 bits instead of the first 5. For this helper, I got the table from the `N=64` case.